orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/aws/ApplySourceServerGroupCapacityTask.groovy

```diff
@@ -80,6 +82,26 @@ class ApplySourceServerGroupCapacityTask extends AbstractServerGroupTask {
     }
   }
 
+  @Override
+  Moniker convertMoniker(Stage stage) {
```

Comment:
Confused as to how this actually works ... you've got two `return` statements in the same scope?

Anyhow ... this stage should be applying the previously captured capacity to the newly created server group. 

Generally speaking, I believe it should be applicable across cloud providers and certainly must be used if `CaptureSourceServerGroupCapacityTask` is used (as this task pins min=desired capacity).

Reply:
Ah, thank you for pointing that out. The first `return` was left over from debugging. It would still work, but is not the intended behavior. 

The intended behavior is to grab the target sever group that will be used in the resize operation. Then return its moniker. Since we are getting the target server group from Clouddriver, it is guaranteed to have a moniker.